### ISSUE TITLE
Error loading login page of "Instagram"

### ISSUE THREAD

**Issue by allankiezel** [nraboy/ng2-cordova-oauth#78]
I recently started getting an error with the instagram login. The window opens as expected and shows the instagram login form, however once entering my credentials and hitting Log in the window immediately closes before the authorization step and the error is: Error loading login page of "Instagram". Has anyone else run into this recently? It seems to happen if you've already approved the app.

**Comment by KevtechApps** [comment]
yes just started getting a similar issue on ios. The instagram signin page loads but then in xcode console
webView:didFailLoadWithError - -999: The operation couldn’t be completed. 
(NSURLErrorDomain error -999.)

**Comment by filipnaumoski** [comment]
I have same problem here.

**Comment by alextomas80** [comment]
I have same problem
